packages/strapi-plugin-upload/admin/src/components/CardPreview/index.js

```diff
@@ -25,7 +33,10 @@ const CardPreview = ({ extension, hasError, hasIcon, url, previewUrl, type, with
   if (isFile) {
     return (
       <Wrapper isFile>
-        <FileIcon ext={extension} />
+        <FileWrap>
+          <FileIcon ext={extension} />
+          {filename && <p>{filename}</p>}
```

Comment:
I saw that you declared a `''` as a default prop for _filename_, would that mean this condition will always return true? :-)

Reply:
No, since an empty string in javascript is falsy.